pkgs/applications/blockchains/masari/default.nix

```
@@ -1,25 +1,35 @@
 { lib, stdenv, fetchFromGitHub, cmake, pkg-config, unbound, openssl, boost
-, lmdb, miniupnpc, readline }:
+, lmdb, miniupnpc, readline, git, zeromq, libsodium, rapidjson, cppzmq }:
 
 stdenv.mkDerivation rec {
-  pname = "masari";
-  version = "0.1.4.0";
+  name = "masari";
+  version = "latest-2022-10-09";
```

We use the `unstable-` prefix
```suggestion
  version = "unstable-2022-10-09";
```